src/js/common/pages/Politician/PoliticianDetailsPage.jsx

```diff
@@ -1069,7 +1069,11 @@ class PoliticianDetailsPage extends Component {
                       </div>
                     </AboutAndEditFlex>
                     {politicianDescription ? (
-                      <ReadMore numberOfLines={6} textToDisplay={politicianDescription} />
+                      <ReadMore numberOfLines={6}
+                      textToDisplay={politicianDescription}
+                      eventName="clickShowMoreAboutPolitician"
+                      politicianIdShowMore={PoliticianDetailsPage.politicianWeVoteIdForDisplay}
+                      politicianNameShowMore={politicianName}/>
```

Comment:
The `/>` should go on its own line, and have the same indention as the opening `<ReadMore`. Please remove politicianNameShowMore.